desk/app/chat.hoon

```diff
@@ -41,10 +44,14 @@
       abet:init:cor
     [cards this]
   ::
-  ++  on-save  !>(state)
+  ++  on-save  !>(state(old-epic our-epic))
   ++  on-load
     |=  =vase
     ^-  (quip card _this)
+    ?:  =(1 2)
```

Comment:
Is this just a guard until we enable this?

Reply:
yes